docroot/themes/humsci/humsci_basic/templates/menus/macros/nav-menu.twig

```diff
@@ -27,9 +27,26 @@
     {% endif %}
 
     <li{{ list_attributes }}>
-      <a{{ link_attributes }}>{{ item.title }}</a>
-      {% if item.below %}
-        {{ menus.nav_menu(item.below, menu_level + 1, class_prefix) }}
+      {% if item.url.toString() is empty %}
+        {% set link_attributes = link_attributes.addClass('hb-nested-toggler') %}
+        <a{{ link_attributes }} id="{{ item.title|replace({' ': ''}) }}{{ menu_level }}">{{ item.title }}</a>
```

Comment:
should we add role=button?

Reply:
I added the role and also adjusted the JS to make sure that the spacebar will also toggle the anchor tags that are buttons.